docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -1,8 +1,18 @@
 # Operating a managed Cluster
 
-## Upgrade the Cluster
+The `spec.topology` field added to the Cluster object as part of ClusterClass allows changes made on the Cluster to be propagated across all relevant objects. This means the Cluster object can be used as a single point of control for making changes to the Cluster topology. 
 
-The `spec.topology` field added to the Cluster object as part of ClusterClass allows changes made on the Cluster to be propagated across all relevant objects. This turns a Kubernetes cluster upgrade into a one-touch operation.
+A managed Cluster can be used to:
+* [Upgrade a Cluster](#upgrade-a-cluster)
+`* [Scale a MachineDeployment](#scale-a-machinedeployment)
+* [Scale a ControlPlane](#scale-a-controlplane)
+* [Add a MachineDeployment](#add-a-machinedeployment)
+* [Rebase a Cluster to a different ClusterClass](#rebase-a-cluster)
+* [Use variables in a Cluster](#use-variables)
```

Comment:
```suggestion
* [Scale a ControlPlane](#scale-a-controlplane)
* [Scale a MachineDeployment](#scale-a-machinedeployment)
* [Add a MachineDeployment](#add-a-machinedeployment)
* [Changing Cluster variables](#use-variables)
* [Rebase a Cluster to a different ClusterClass](#rebase-a-cluster)
```
Suggesting a slightly different order so we have examples about CP, then all the examples about MD, then the more advanced use cases

Reply:
I'd prefer to keep the CP below the MD examples as I feel like they're the more common usages, but I'll group the MDs at the top, followed by CP, then variables and rebase.